backend/package.json

```
@@ -11,6 +11,10 @@
     "lint:fix": "eslint src/ --fix",
     "format": "prettier --write src/",
     "format:check": "prettier --check src/",
+    "docs": "jsdoc -c jsdoc.json",
+    "docs:validate": "echo 'JSDoc validation is now handled by ESLint - use npm run lint'",
```

Biome (2.1.2)

[error] The key docs:validate was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)